ksql-engine/src/test/java/io/confluent/ksql/codegen/SqlToJavaVisitorTest.java

```diff
@@ -330,6 +333,134 @@ public void shouldGenerateCorrectCodeForDecimalMod() {
     assertThat(java, is("(TEST1_COL8.remainder(TEST1_COL8, new MathContext(2, RoundingMode.UNNECESSARY)).setScale(1))"));
   }
 
+  @Test
+  public void shouldGenerateCorrectCodeForDecimalDecimalEQ() {
+    // Given:
+    final ComparisonExpression compExp = new ComparisonExpression(
+        Type.EQUAL,
+        new QualifiedNameReference(QualifiedName.of("TEST1.COL8")),
+        new QualifiedNameReference(QualifiedName.of("TEST1.COL8"))
```

Comment:
Perhaps make these two distinct so the tests check ordering of the arguments as well? (Same comment for all decimal-decimal tests in this file.)